### ISSUE TITLE
Enhancement to Ownership Information Field

### ISSUE THREAD

**Issue by martinhannigan** [peeringdb/peeringdb#1115]
**Is your feature request related to a problem? Please describe.**

Enhancement. In the Property Owner option there are two choices. Owner and Lessee. Owner is an excellent, plain english, description of state. However, Lessee is not. While it is legitimately descriptive, it can be confusing and its meaning is not obvious. Should update tooltip to map as well.

**Who is affected by the problem?**

Everyone.

**What is the impact?**

Minor confusion easily rectified by stopping what you are doing and using your favorite search engine. imprecise tooltip description.

**Are there security concerns?**

No

**Are there privacy concerns?**

No

**Describe the solution you'd like**

Change the Lessee selection to "Leased or Rented"

Change the tooltip to:
"Leasing or renting is an agreement with a property owner for use of the property."

**Do you think this feature will require a formal design?**
No.

**Describe alternatives you've considered**

I considered doing nothing. However, I abandoned that since usability is key.

**Could this feature request need support from the Admin Committee?**

Perhaps. 

**What is the proposed priority?**

Low priority.

**Provide a rationale for any/all of the above**

<img width="244" alt="Screen Shot 2022-02-16 at 3 02 38 PM" src="https://user-images.githubusercontent.com/21081154/154356603-e7728428-8814-4db9-a88d-5199cfa7d157.png">

**Additional context**

**Comment by leovegoda** [comment]
This looks like a really simple change to make once we have agreement on the language

**Comment by arnoldnipper** [comment]
Let's not make cosmetic changes while we are working on a general solution #1110

**Comment by martinhannigan** [comment]
There's nothing for me to do here. It should be included on the list for
action at whatever point the product manager determines is proper.

**Comment by martinhannigan** [comment]
Where is this issue? Open for awhile.

**Comment by leovegoda** [comment]
It is still in Decide because no @peeringdb/pc members have +1'd it. Arnold wanted to wait until `campus` was deployed (and it is) but no-one else has commented

**Comment by martinhannigan** [comment]
+1

**Comment by ynbrthr** [comment]
+1

**Comment by grizz** [comment]
+1

**Comment by peterhelmenstine** [comment]
+1

**Comment by martinhannigan** [comment]
Id suggest we review design concept prior to execution.

**Comment by leovegoda** [comment]
@martinhannigan Did you mean to close this? We've not yet scheduled this work.

**Comment by martinhannigan** [comment]
No, and apologies. I believe you are usually responsible for that. If it
happened, it was a keyboard error. Apologies!

**Comment by martinhannigan** [comment]
Doesn't appear to map back to issue as stated.

**Comment by europe-weighed** [comment]
IMPLEMENTERA DEL ALL ALIAS This command configures git to use GitHub CLI as a credential helper. For more information on git credential helpers please reference: https://git-scm.com/docs/gitcredentials.

By default, GitHub CLI will be set as the credential helper for all authenticated hosts. If there is no authenticated hosts the command fails with an error.

Alternatively, use the --hostname flag to specify a single host to be configured. If the host is not authenticated with, the command fails with an error.

Options
-f, --force <--hostname>
Force setup even if the host is not known. Must be used in conjunction with --hostname
-h, --hostname <string>
The hostname to configure git for
Examples
# Configure git to use GitHub CLI as the credential helper for all authenticated hosts
$ gh auth setup-git

# Configure git to use GitHub CLI as the credential helper for enterprise.internal host
$ gh auth setup-git --hostname enterprise.internal
See also



This command configures git to use GitHub CLI as a credential helper. For more information on git credential helpers please reference: https://git-scm.com/docs/gitcredentials.

By default, GitHub CLI will be set as the credential helper for all authenticated hosts. If there is no authenticated hosts the command fails with an error.

Alternatively, use the --hostname flag to specify a single host to be configured. If the host is not authenticated with, the command fails with an error.

Options
-f, --force <--hostname>
Force setup even if the host is not known. Must be used in conjunction with --hostname
-h, --hostname <string>
The hostname to configure git for
Examples
# Configure git to use GitHub CLI as the credential helper for all authenticated hosts
$ gh auth setup-git

# Configure git to use GitHub CLI as the credential helper for enterprise.internal host
$ gh auth setup-git --hostname enterprise.internal
See also
